site/content/en/docs/Getting started/linux.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Linux"
+linkTitle: "Linux"
+weight: 1
+description: >
+  How to install and start minikube on Linux.
```

Comment:
I know the description is good for the title but visually doesnt make sense, since it gets repeated 4 times in the front page. I suggest a better title and no descrption
maybe
"Try on Linux"